charts/observability-pipeline/templates/beekeeper-deployment.yaml

```diff
@@ -47,10 +65,6 @@ spec:
               value: {{ .Values.beekeeper.pipelineInstallationID }}
             - name: HONEYCOMB_MGMT_API_KEY
               value: {{ .Values.beekeeper.publicMgmtKey }}
-            {{- if .Values.beekeeper.telemetry.enabled }}
```

Comment:
Is there any downside for removing this condition to only enable the env var if telemetry is enabled?